portality/templates/formcontext/maned_journal_review.html

```diff
@@ -141,4 +141,45 @@ <h3 class="heading-aligned-with-fields">Old journal information</h3>
 <script type="text/javascript">
     var notes_deletable = true;
 </script>
+
+<script>
+// When the editor group field is changed, refresh the options for editor
+$("#editor_group").change(function() {
```

Comment:
hmm, factoring this out into a static file isn't as easy as I'd hoped, because I rely on defining the script in the template to use ```url_for()``` as part of my ajax request.